hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -256,6 +256,117 @@ def hpat_pandas_series_values_impl(self):
     return hpat_pandas_series_values_impl
 
 
+@overload_method(SeriesType, 'value_counts')
+def hpat_pandas_series_value_counts(self, normalize=False, sort=True, ascending=False, bins=None, dropna=True):
+    """
+    Pandas Series method :meth:`pandas.Series.value_counts` implementation.
+    .. only:: developer
+
+       Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_value_counts*
```

Comment:
Please add flag `-k` to be able to run tests via this command:
`python -m hpat.runtests -k hpat.tests.test_series.TestSeries.test_series_value_counts*`